packages/zendesk-adapter/src/config.ts

```diff
@@ -1688,6 +1692,22 @@ export const DEFAULT_TYPES: ZendeskApiConfig['types'] = {
       },
     },
   },
+  [ARTICLE_ATTACHMENT_TYPE_NAME]: {
+    transformation: {
+      idFields: ['&brand', 'filename'],
```

Comment:
missed this earlier - we should use the article id here, otherwise we'll have conflicts on filenames (discussing on slack as well, just writing here so we don't miss it)

Reply:
reopening as it doesn't look like this was addressed - we should make this a child of the parent, so probably
```suggestion
      idFields: ['filename'],
      extendsParentId: true,
```
and make sure the article is its parent so that the id gets re-generated correctly in the `reference_id_fields` filter